src/app/archive/cli/archive_cli.ml

```diff
@@ -38,38 +38,26 @@ let command_run =
            "int Delete blocks that are more than n blocks lower than the \
```

Comment:
*reviewer notes*

remove the Runtime_config.Json_loader, and add back the genesis constants and constraint constants. In the archive config we will now simply load the genesis and constraint configs from the runtime.

Reply:
we also propogate the runtime config instead of the precomputed values, opting for a more bundled design pattern with the config. This seems okay to me, despite us having to do more runtime checking and parsing at lower levels. We potentially sacrifice a missed check in exchange for a more accessible api.